src/geyser/core.zig

```diff
@@ -170,25 +173,19 @@ pub const GeyserWriter = struct {
         const total_len = bincode.sizeOf(payload, .{});
 
         // obtain a memory to write to
-        self.io_free_fba.mux.lock();
```

Comment:
+1 to improving this. I found it a bit clunky when I made the initial switch to the recycle allocator